website/server/controllers/api-v3/tasks/groups.js

```diff
@@ -200,13 +200,25 @@ api.assignTask = {
     if (canNotEditTasks(group, user, assignedUserId)) throw new NotAuthorized(res.t('onlyGroupLeaderCanEditTasks'));
 
     let promises = [];
-
-    if (user._id !== assignedUserId) {
-      const taskText = task.text;
-      const managerName = user.profile.name;
-
+    const taskText = task.text;
+    const userName = user.profile.name;
+
+    if (user._id === assignedUserId) {
+      const managerIds = Object.keys(group.managers);
+      managerIds.push(group.leader);
+      const managers = await User.find({_id: managerIds}, 'notifications').exec();
+      managers.forEach((manager) => {
+        if (manager._id === user._id) return;
+        manager.addNotification('GROUP_TASK_CLAIMED', {
+          message: res.t('taskClaimed', {userName, taskText}),
```

Comment:
this will add the notification in the language of the user not of the manager

Reply:
Hmm, can I pass the manager's `preferences.language` as an argument? I _think_ so from looking at `i18n.t` but not 100% sure...

Reply:
Yeah let's use `manager. preferences.language `